src/services/api.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+const api = import.meta.env.VITE_API;
+
+const endpoint = {
+  USERS: 'users',
+  TICKET_TYPES: 'ticket_types',
+  RESERVATION_STATUSES: 'reservation_statuses',
+  SEANCES: 'seances',
+  MOVIES: 'movies',
+  TICKET_DESKS: 'ticket_desks',
+  HALLS: 'halls',
+  MOVIE_GENRES: 'movie_genres',
+  TICKETS: 'tickets',
+  RESERVATIONS: 'reservations',
+};
```

Comment:
There is probably a redundancy in this object since I made it based on API's admin panel endpoints, not API's documentation, so if I'm correct, please ignore that for a moment.